pkg/kube/structured/structured.go

```diff
@@ -210,6 +210,17 @@ func DeploymentIsRunning(kubeClientset kubernetes.Interface, name, namespace str
 	return nil
 }
 
+func ConfigMapDataExists(kubeClientset kubernetes.Interface, configMapName, namespace, key, value string) error {
```

Comment:
Lets make `ConfigMapDataExists` be something like `ConfigMapDataHasKeyAndValue` or `ConfigMapDataHasKey`.